README.md

```diff
@@ -37,12 +37,20 @@ from App Runner services.
 
 ## Labels
 
-* **container**: Container associated with a metric.
-* **cpu**: Available to CPU metrics, helps to breakdown metrics by CPU.
-* **device**: Network interface device associated with the metric. Only
+### On task-level metrics
+* **task_arn**: [ARN of the task](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticcontainerservice.html#amazonelasticcontainerservice-resources-for-iam-policies) associated with a metric.
+* **family**: Task definition family associated with a metric.
+* **revision**: Revision of the task definition family associated with a metric.
+
+### On container-level metrics
+
+* **container**: Name of the container (as in the ECS task definition) associated with a metric.
+* **image**: Docker image identifier (e.g. `name:tag`, `name@digest`) of the container.
+* **interface**: Network interface device associated with the metric. Only
   available for several network metrics.
 
 ## Example output
+TODO update
```

Comment:
I'll defer doing this until the rest of the PR is approved, as the contents of this example are sensitive to even minor changes in the PR.

Reply:
Done.  A full before/after diff can be found here: https://github.com/isker/ecs-exporter-cdk/tree/master/experiments/overhaul.